File: app/src/main/java/com/wujia/intellect/terminal/mvp/login/LockActivity.java
package com.wujia.intellect.terminal.mvp.login;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

import com.wujia.intellect.terminal.R;
import com.wujia.lib_common.base.BaseActivity;

public class LockActivity extends BaseActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_lock);
    }

    @Override
    protected int getLayout() {
        return 0;
    }

    @Override
    protected void initEventAndData(Bundle savedInstanceState) {

    }
}

File: app/src/main/java/com/jingxi/smartlife/pad/mvp/login/LockActivity.kt
package com.jingxi.smartlife.pad.mvp.login

import android.os.Bundle

import com.jingxi.smartlife.pad.R
import com.wujia.lib_common.base.BaseActivity
import kotlinx.android.synthetic.main.activity_lock.*

class LockActivity : BaseActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
    }

    override fun getLayout(): Int {
        return R.layout.activity_lock
    }

    override fun initEventAndData(savedInstanceState: Bundle) {
        lock_img_bg!!.setOnClickListener { }
    }
}
